src/main/java/mate/academy/service/UserService.java

```diff
@@ -8,13 +8,22 @@ public class UserService {
             new User("alice@i.ua", "1234")
     };
 
+    public static User[] getUsers() {
```

Comment:
remove this method, use the method below for login method logic and don't use static, it's a bad practice